pkg/virt-api/webhooks/validating-webhook/validating-webhook_test.go

```diff
@@ -991,6 +993,44 @@ var _ = Describe("Validating Webhook", func() {
 	})
 
 	Context("with VirtualMachineInstance spec", func() {
+		It("should accept valid machine type", func() {
```

Comment:
There's a lot more test cases than just the network-config tests being added here. Is this intentional? I reviewed them and they look good, it was just unexpected to see these additional validation tests in this patch.

Reply:
Once i saw coveralls I felt the need to go green... I was already adding some missing tests so I figured why not a few more?